recipe/remove-debug-symbols.patch

```diff
@@ -0,0 +1,10 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -187,6 +187,7 @@
+     add_compile_options( -pthread )
+ 
+     add_compile_options("$<$<CONFIG:RELWITHDEBINFO>:-O2>")
++    add_compile_options("$<$<CONFIG:RELWITHDEBINFO>:-g0>")
```

Comment:
@msoos expressed concerns about `Release` not having `assert`s. So, this is a hacky workaround to use `RelWithDebugInfo` instead of `Release` but still not have the library inflated by debug symbols (reducing file size by factor 16 or so, IIRC). If no symbols at all seems inappropriate, we can use `-g1` here, which will increase the size by 60-70 %, IIRC.

Reply:
Seems reasonable to me.